docker/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM ubuntu:bionic
+
+RUN apt-get update && apt-get install -y git curl imagemagick librsvg2-bin gnupg && \
+    curl -sL "https://keybase.io/crystal/pgp_keys.asc" | apt-key add - && \
+    echo "deb https://dist.crystal-lang.org/apt crystal main" | tee /etc/apt/sources.list.d/crystal.list && \
+    apt-get update && apt-get install -y crystal libyaml-dev libxml2-dev && \
+    git clone https://github.com/omarroth/invidious
```

Comment:
Would it be possible to use the local repository instead of cloning it? I think that would make it easier to test changes, as well as making it easier for someone to run off a fork of this project.

Reply:
There are several ways how the docker image can be constructed:
1. for production it should be built using a git clone from a tagged master revision
2. for dev it should be build using the local sources, which can be either
    - included in the image using dockerfile ADD instruction - so you'd need to rebuild the image after a change in the source
    - or mounted using volume - however this would also need some automatic way to recompile the sources, it's useful mainly for scripted languages. I'm not very familiar with compiled languages

I will change it to the ADD instruction, we'll see how well it works